README.md

```diff
@@ -91,6 +91,11 @@ Basically, the GUI is a serverless Angular application which use the Backend JSO
 
 Other than the installation file, the [docs directory](docs/), contains a translation guide for the GUI
 
+#### Security of our dependencies
```

Comment:
This is a really good clarification!
It should be checked at least for every release, right? Could you make a PR to update the ReleaseProcess document in the main repo?

Reply:
Every time we update Perl!
But yes, ideally, at least for every release.